clients/csrf/csrf_paramters/csrf_parameters.go

```diff
@@ -0,0 +1,6 @@
+package csrf_paramters
+
+type CsrfParameters struct {
```

Comment:
The whole CSRF thing is a request header. Its values are header name and header value

Reply:
Renamed to -> CsrfRequestHeader